src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { SvgSquare } from '$particles/svg-square'
+
+export const WaffledSquare = ({
+  name = 'Germany',
```

Comment:
I think we should try and avoid having components know about the data they are being used with. The value `Germany` might make sense for the EU election page, but if we were to reuse this component somewhere else it wouldn't.